data/linux-bridge/003-bridge-marker-rbac.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: bridge-marker-cr
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - nodes
+  - nodes/status
+  verbs:
+  - get
+  - update
+  - patch
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: bridge-marker-crb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: bridge-marker-cr
+subjects:
+- kind: ServiceAccount
+  name: bridge-marker
+  namespace: kube-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: bridge-marker
+  namespace: kube-system
+  
```

Comment:
drop empty line, add new line to the end of the file

Reply:
done

Reply:
I still see the same problem here.
Can you please take a look ?

Reply:
There is a newline at the end of the file. Should this be dropped?

Reply:
now it looks good thanks! :)